Index: src/Makefile.am
===================================================================
--- src/Makefile.am	(revision 5895)
+++ src/Makefile.am	(working copy)
@@ -27,6 +27,11 @@
 # The directory in which to install headers
 libMOAB_la_includedir = $(includedir)

+if HAVE_CGM
+  AM_CPPFLAGS += $(CGM_INCLUDES) $(CGM_CPPFLAGS) -DCGM $(MOAB_CGM_DEFINES)
+  AM_CXXFLAGS += $(CGM_CXXFLAGS)
+  libMOAB_la_LIBADD += $(CGM_LIBS)
+endif

 # The list of source files, and any header files that do not need to be installed
 libMOAB_la_SOURCES = \
